crates/bevy_asset/src/loader.rs

```diff
@@ -288,7 +288,7 @@ impl<A: Asset> AssetContainer for A {
     }
 }
 
-/// An error that occurs when attempting to call [`LoadContext::load_direct`]
+/// An error that occurs when attempting to call [`DirectNestedLoader::load`](crate::DirectNestedLoader::load)
```

Comment:
I didn't find any `LoadContext::load_direct`, the closest thing where this is used seemed to be `DirectNestedLoader::load` but I'm not sure it's correct.